addons/mail/static/src/js/chatter.js

```diff
@@ -111,6 +112,16 @@ var Chatter = Widget.extend({
     // Public
     //--------------------------------------------------------------------------
 
+    /**
+     * resets _suggestedPartnersProm and call _openComposer
+     * if composer is already open, to update suggested recipients
+     */
+    resetSuggestedPartners() {
+        this._resetSuggestedPartners();
```

Comment:
no private method. Resetting suggested partners should always re-render suggested recipients on composer if necessary.

Reply:
I will keep ```_resetSuggestedPartners``` private method because it just resets ```_suggestedPartnersProm``` to ```undefined``` in ```_seState``` we just want to reset ```_suggestedPartnersProm``` as chatter is going to destroy when record changes